ios/Rainbow.xcodeproj/project.pbxproj

```diff
@@ -754,6 +755,7 @@
 				DEBUG_INFORMATION_FORMAT = "dwarf-with-dsym";
 				DEVELOPMENT_TEAM = L74NQAQB8H;
 				ENABLE_BITCODE = NO;
+				"EXCLUDED_ARCHS[sdk=*]" = arm64;
```

Comment:
Can you try removing this line and running the tests in release mode? I don't want to make this change for the release builds.

Reply:
switched to Simulator only